tests/components/camera/test_mqtt.py

```diff
@@ -0,0 +1,28 @@
+"""The tests for mqtt camera component."""
+import asyncio
+
+from homeassistant.setup import async_setup_component
+import homeassistant.components.mqtt as mqtt
+
+
+@asyncio.coroutine
+def test_run_camera_setup(hass, test_client):
```

Comment:
It seems that this test is flaky: https://travis-ci.org/home-assistant/home-assistant/jobs/222640657#L286

Instead of depending on the built-in MQTT broker the test should have mocked the MQTT broker.